test/unit/input_template_renderer_test.rb

```diff
@@ -288,7 +288,11 @@
           end
 
           context 'with existing variable implemented as smart variable' do
-            let(:puppet_class) { FactoryGirl.create(:puppetclass, :environments => [environment], :hosts => [renderer.host]) }
+            let(:puppet_class) do
```

Comment:
I had to do this because the host_classes created by the old factory would not validate, because hosts on Puppetclass is not mass_assignable on Foreman anymore.